websites/G/Google Messages/tsconfig.json

```diff
@@ -1,6 +1,6 @@
-{
-    "extends": "../../../tsconfig.json",
-    "compilerOptions": {
-      "outDir": "./dist/"
-    }
+{
+    "extends": "../../../tsconfig.json",
+    "compilerOptions": {
+      "outDir": "./dist/"
+    }
   }
```

Comment:
why you edit another presence tsconfig? xD

Reply:
Not sure why it was edited 😅

Reply:
Seems like I'm just dumb I don't know how to undo this plus the file looks just same